Automattic/wp-calypso
# A4A: Implement VIP hosting page v3.

This PR implements the new WPCOM hosting section v3 (feature list, testimonials & CTA section).

<img width="725" alt="Screenshot 2024-12-10 at 6 33 10 PM" src="https://github.com/user-attachments/assets/f2c82d6c-bcfe-47f5-b983-351ccb05404b">


Depends on https://github.com/Automattic/wp-calypso/pull/97230
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1556
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1557
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1558

## Proposed Changes

* Implement the 'Feature list', 'Testimonials', and 'CTA' sections.

## Why are these changes being made?

* This is part of the In-product hosting page v3 project.


## Testing Instructions

* Use the A4A live link and go to the `/marketplace/hosting/vip` page.
* Confirm you see the new sections.

## Pre-merge Checklist
